<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SqlDatasetEditor</class>
 <widget class="QWidget" name="SqlDatasetEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>718</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Database:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="databaseField"/>
     </item>
     <item>
      <widget class="QPushButton" name="bLoadFile">
       <property name="text">
        <string>File...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bExtended">
       <property name="text">
        <string>Extended Options</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Host:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="hostField"/>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>User:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3" colspan="2">
       <widget class="QLineEdit" name="userField"/>
      </item>
      <item row="0" column="5" colspan="2">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Password:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="7" colspan="2">
       <widget class="QLineEdit" name="passwordField"/>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="driver"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Driver:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Available Drivers:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="7">
       <widget class="QComboBox" name="driversList"/>
      </item>
      <item row="1" column="8">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>224</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="4">
       <widget class="QPushButton" name="bCopyDriverName">
        <property name="text">
         <string>&lt;&lt;</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="TextEdit" name="textEdit"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TextEdit</class>
   <extends>QTextEdit</extends>
   <header>sqldatasethelper.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
